skillbridge/server/python_server.py

```diff
@@ -1,13 +1,22 @@
+from os import environ, unlink, getenv, path
+from socketserver import StreamRequestHandler, ThreadingMixIn, BaseServer
+from socketserver import BaseRequestHandler
+from logging import getLogger, basicConfig, WARNING
+from sys import stdout, stdin, argv, platform, stderr
 import logging
 from argparse import ArgumentParser
 from logging import WARNING, basicConfig, getLogger
-from os import unlink
+from os import unlink, environ
```

Comment:
It seems that something went wrong here with the imports several modules are imported twice.

flake8 should also give a warning